From c3942a1d0fc2dda1ee2b5108e1feb772abe4bec1 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Wed, 28 May 2025 16:43:23 -0400
Subject: [PATCH 071/156] winebus: PROTON_PREFER_SDL -> PROTON_USE_SDL

More accurately name the env
---
 dlls/winebus.sys/bus_udev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/winebus.sys/bus_udev.c b/dlls/winebus.sys/bus_udev.c
index d1c3109c13e..9c992d84a64 100644
--- a/dlls/winebus.sys/bus_udev.c
+++ b/dlls/winebus.sys/bus_udev.c
@@ -1646,7 +1646,7 @@ static void udev_add_device(struct udev_device *dev, int fd)
 
     if (desc.vid == 0x28de && desc.pid == 0x11ff && !strcmp(subsystem, "input"))
     {
-        char *env = getenv("PROTON_PREFER_SDL");
+        char *env = getenv("PROTON_USE_SDL");
         if (env && !strcmp(env, "1"))
         {
             TRACE("evdev %s: gnoring steam input virtual controller\n", debugstr_a(devnode));
-- 
2.49.0

